code/modules/clothing/under/syndicate.dm

```diff
@@ -18,6 +18,7 @@
 	item_state = "bl_suit"
 	worn_state = "tactifool"
 	siemens_coefficient = 1
+	has_sensor = 1

```

Comment:
The tacticool turtleneck isn't supposed to have sensors, this has been an intentional feature for a while due to the fact that the tacticool turtleneck gives 10 armor protection.

Reply:
This variant does not have armor.

Reply:
Essential what lok said. This is the civilian loadout version